### ISSUE TITLE
TypeError in wtg file creation using LayoOWts PowerCurve

### ISSUE THREAD

**Issue by jnteruel** [GTornero/layOWt#9]
I've come across the following _**TypeError**: cannot serialize None (type NoneType)_ when trying to export to a wtg file a power curve from the "internal" database. Adding a copy of the code which rises error below for convenience.

![TypeError](https://user-images.githubusercontent.com/75723200/191271403-58cb311d-b480-49be-bac1-eecb9f6336e3.PNG)

from layowt.wtg import PowerCurve

```
engine = PowerCurve.database_engine()
with engine.connect() as conn:
    # PCinternals = PowerCurve.get_internal_turbines(engine=conn)
    pc = PowerCurve.from_database("VESTAS_NGT260_20000_GEN_1.225_0.06_003", engine=conn)
pc.set_hub_height(160)
pc.to_wtg()
```
It is caused by the fact that the attribute "ref_document_id" is set to **None**. I found a patch by adding the following line before running *pc.to_wtg()*:

pc.attributes["ref_document_id"] = "No Ref Document"

**Comment by jnteruel** [comment]
Yes, in fact, adding the following line before saving fixes issue: 
from layowt.wtg import PowerCurve

```
engine = PowerCurve.database_engine()
with engine.connect() as conn:
    # PCinternals = PowerCurve.get_internal_turbines(engine=conn)
    pc = PowerCurve.from_database("VESTAS_NGT260_20000_GEN_1.225_0.06_003", engine=conn)
pc.set_hub_height(160)
**pc.attributes["ref_document_id"] = "No Ref Document"**
pc.to_wtg()
```

**Comment by GTornero** [comment]
Also need to apply the same fix for when the release date field is empty.

**Comment by GTornero** [comment]
also fix this for no revision numbers

**Comment by GTornero** [comment]
Issues fixed in version 0.2.0
